device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,54 @@
+
+{%- set default_cable = '300m' %}
+
+{%- set ports2cable = {
+        'torrouter_server'       : '300m',
```

Comment:
is this intentional?

Reply:
Good catch. This is problem copied from source file. I'll create another PR to address the other device.